### ISSUE TITLE
Page is rendered twice when going back from external page on safari with basePath and assetPrefix set on next.config.js

### ISSUE THREAD

**Issue by edwin-tandiono** [vercel/next.js#21408]
**What version of Next.js are you using?**

10.0.6-canary.6

**What version of Node.js are you using?**

10.23.0

**What browser are you using?**

Safari

**What operating system are you using?**

macOS

**How are you deploying your application?**

next start

**Describe the Bug**

Going back to the next app from external site causes _app.jsx to render twice on Safari. Testing it on Chrome renders it only once. It seems to only happen when basePath and assetPrefix is set in next.config.js.

Notes: I tried on next 9.5.4 and 9.5.1, the same thing happened.

**Expected Behavior**

Should only render once when going back from Safari.

**To Reproduce**

I've made a simple codesandbox to test it https://codesandbox.io/s/ios-back-button-42frx?file=/README.md.
1. Open this app on Safari: https://42frx.sse.codesandbox.io/test -> should console logs 'RENDER' once
2. Go to external site, e.g. https://www.google.com
3. Click back button
4. Should open out app again, but console logs 'RENDER' twice

**Comment by lednhatkhanh** [comment]
I got this problem as well even without settings any of the path above on safari 13.1, it's working fine on safari 14 though.

**Comment by lednhatkhanh** [comment]
@echo-delta Have you found a solution for this? Thank you.

**Comment by echo-delta** [comment]
Unfortunately no, for now I refactored my code to not depend on the render process as a workaround.

**Comment by lednhatkhanh** [comment]
That’s also what I’m doing now, really hope that this will be fixed.

**Comment by balazsorban44** [comment]
I believe this has been fixed here: https://github.com/vercel/next.js/pull/32687

if not, please open a new bug report with a reproduction and the newest version of Next.js

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
